index.html

```diff
@@ -3679,8 +3683,8 @@ <h4>Security Considerations</h4>
         MUST be adopted by compliant implementations.</span>
     </p>
     <p class="note">
-      Please see <a href="https://w3c.github.io/wot-security-best-practices/">
-        WoT Security Best Practices</a> for implementation advice.
+      Please see <a href="https://www.w3.org/TR/wot-security/">
```

Comment:
Sorry for some confusion caused. I mean this exact line with the reference as a link is not needed. Of course, if it is in xref database, we can just use it.